src/base_circuitpython/displayio/group.py

```diff
@@ -63,6 +63,39 @@ def __init__(
         self.__parent = None
         self.__hidden = False
 
+    @property
+    def x(self):
+        return self.__x
+
+    @x.setter
+    def x(self, val):
+        changed = val != self.__x
+        self.__x = val
+        if changed:
+            self.__elem_changed()

```

Comment:
_&ast;nit&ast;_
Can streamline the setters a bit by omitting `changed` like this:
```
def x(self, val):
  if self.__x != val:
    self.__x = val
    self.__elem_changed()
```
Up to you!

Reply:
Ahh, true true!!